src/test/java/reactor/pool/CommonPoolTest.java

```diff
@@ -1178,6 +1179,22 @@ void disposeLaterReleasedConcurrently(Function<PoolBuilder<Integer, ?>, Abstract
 		          .untilAtomic(live, CoreMatchers.is(0));
 	}
 
+	@ParameterizedTest
+	@MethodSource("allPools")
+	void allocatorErrorInAcquireDrains(Function<PoolBuilder<String, ?>, AbstractPool<String>> configAdjuster) {
+		PoolBuilder<String, ?> builder = PoolBuilder
+				.from(Mono.delay(Duration.ofMillis(100))
```

Comment:
how about to use an Atomic to alternate between an error and a successful allocation, and validate that two concurrent `acquire` will see the second one succeed ?

Reply:
done